src/screens/home/HomeScreen.js

```diff
@@ -1,17 +1,26 @@
 import React, { useState } from 'react'
 import { StatusBar, View } from 'react-native'
+import Minilog from '@cozy/minilog'
+
+const konnLog = Minilog('Konnector')
 
 import DebugView from '/screens/connectors/DebugView'
 import HomeView from '/screens/home/components/HomeView'
 import LauncherView from '/screens/connectors/LauncherView'
 import { StatusBarStyle } from '/libs/intents/setFlagshipUI'
 import { styles } from '/screens/home/HomeScreen.styles'
+import { useConnectorState } from 'src/providers/ConnectorState/ConnectorStateProvider'
 
 export const HomeScreen = ({ route, navigation }) => {
   const [barStyle, setBarStyle] = useState(StatusBarStyle.Light)
   const [debug] = useState(false)
   const [launcherContext, setLauncherContext] = useState({ state: 'default' })
+  const { addLog } = useConnectorState()
 
+  const onKonnectorLog = async logObj => {
+    konnLog[logObj.level](`${logObj.slug}: ${logObj.msg}`)
+    addLog(logObj)
+  }
```

Comment:
It seems weird to me, that this is the HomeScreen job to `addLog`. Shouldn't be the responsibility of the `LauncherView`? @Ldoppea

Reply:
This is because we chose to handle logs by the Flagship part (vs Launcher) so we can use the same code to send them on app's restart. Due to this we can use only the Flagship route (the one that does not require the connector's dedicated token).

Also the connector doesn't have to know what is done with the logs. They can be send instantaneously, They can be sent in batch, or not sent due to an error and then sent on next app's start.

So we considered that the responsibility split is more clear if the Launcher only know the `log()` interface and then the Flagship part does all the batch/persist logic.

Reply:
@Ldoppea WDYT about my previous comment?